release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1082,6 +1082,38 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-dev-console-bug-fixes_{context}"]
 ==== Developer Console
 
+* Previously, when a serverless function was created in the create serverless form, BuilldConfig was not created. With this update, if the Pipelines Operator is not installed, if the pipeline resource is not created for particular resource, or if the pipeline is not added while creating a serverless function, it will create BuildConfig as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34143[*OCPBUGS-34143*])
```

Comment:
```suggestion
* Previously, when a serverless function was created in the create serverless form, `BuilldConfig` was not created. With this update, if the Pipelines Operator is not installed, the pipeline resource is not created for particular resource, or the pipeline is not added while creating a serverless function, it will create `BuildConfig` as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34143[*OCPBUGS-34143*])
```